mandatory/3-function-output.js

```diff
@@ -1,15 +1,18 @@
 // Add comments to explain what this function does. You're meant to use Google!
+// The function round Floats  to the nearest whole number using the `Math.round` function
```

Comment:
The function below includes Math.random() not Math.round(), the function will return a random number between 0 and less than 10.